src/main/resources/io/jenkins/plugins/designlibrary/Cards/index.properties

```diff
@@ -1,6 +1,9 @@
-description=Use cards to surface related information and controls to users.
 do.1=Keep your card glanceable, let the user expand your card for more information
 do.2=Make your cards scalable to different screen sizes
 do.3=Do use hierarchy and symbols to add visual context
 dont.1=Don''t add unnecessary visual elements that clutter the card
 dont.2=Don''t have too many actions on one card
+creating-a-card.title=Creating a card
+creating-a-card.description=Cards are designed to automatically expand and fill the available space within their container, making them flexible and easy to use in different layouts. \n\nSince Jenkins doesn''t currently have a built-in grid system, you''ll need to create your own if you want to arrange multiple cards in a grid-like structure. You can do this by utilizing Flexbox or Grid to manage the positioning and spacing of your cards within the parent container.
```

Comment:
Jenkins does have a grid: https://github.com/jenkinsci/jenkins/blob/e3fdfa527e4fefb4b37f04c92a2dd87b8a374b75/war/src/main/webapp/css/responsive-grid.css#L43

(bootstrap 3s grid)

Reply:
Keen to discourage usage of Bootstrap/remnants of it, their class names are sadly really hard to browse for though which makes it a pain to figure out usage levels.

Reply:
not used that much I think but will be used in some places